.github/CODEOWNERS

```diff
@@ -3663,7 +3663,7 @@
 /types/onionoo/                                     @BendingBender
 /types/onoff/                                       @marcel-ernst @Kallu609
 /types/ontime/                                      @Hirse
-/types/open/                                        @Bartvds
+/types/open/                                        @shinnn @SomaticIT @tlent @ffflorian
```

Comment:
I wasn't sure if I actually need to edit this file?

Reply:
It's fine, but if you don't, the weekly run of the `update-codeowners` task will get it.